content/Dev/remote_settings_history.rst

```diff
@@ -0,0 +1,178 @@
+The History of Firefox Remote Settings
+######################################
+
+:tags: python, javascript, rust
+:date: 2020-10-19
+
+
+When I started to write the first lines of this article, it was the last day of Ethan in our team. His departure marked the end of an era that I'll try to tell you about here.
+
+
+Prehistory (~2014)
+==================
+
+While I was working at `Makina Corpus <https://makina-corpus.com>`_, we were regularly coming up with the same kind of architectures: a database, a REST API, and frontend/mobile stuff. Other teams were also starting to consider the idea of «headless CMS».
+
+I wanted to build a reusable backend, where you define your models in JSON and retrieve/store records.
+`Alexis <https://blog.notmyidea.org/>`_ and I wrote down some pseudo code at a DjangoCon and we started the `Daybed project <https://github.com/spiral-project/daybed/>`_, using `Pyramid <https://trypyramid.com/>`_. It became functional and I even talked about it at `FOSDEM <https://archive.fosdem.org/2015/schedule/event/daybed/>`_.
+
+Alexis, `Rémy <https://devhub.io/developer/Natim>`_, and `Tarek <https://ziade.org/>`_ were building Web APIs in the Cloud Services team, and there were needs to offer remote storage features to `Firefox OS <https://en.wikipedia.org/wiki/Firefox_OS>`_ app developers.
+
+I was hired onto their team and we were about to roll out our first app: a «reading list» service (à-la Pocket).
```

Comment:
There is a missing (on purpose?) piece of the bigger picture here.  Mozilla acquired Pocket which effectively killed Reading List.  That left Kinto as a solution without a problem.  At least, that's what stands out in my memory and perspective.

Reply:
I don't think this is accurate, because Readinglist was killed before Kinto existed. And Kinto was being used in Firefox OS if I remember correctly

Reply:
This is actually news to me.  I thought Kinto was built to power Readinglist.